MdeModulePkg/MdeModulePkg.dec

```diff
@@ -658,7 +667,7 @@
   # MU_CHANGE
   # This protocol provides access to Internal Event services that do not need TPL_APPLICATION
   gInternalEventServicesProtocolGuid = {0x7ecd162a, 0xc664, 0x11ec, { 0x9d, 0x64, 0x02, 0x42, 0xac, 0x12, 0x00, 0x02 } }
-
+  

```

Comment:
I noticed some whitespace come into the file in a few places. Can you please try to trim trailing whitespace in all files touched?

Reply:
Fixed the white space changes.